run/opencl/ed25519-donna/ed25519-donna-impl-base.h

```diff
@@ -121,6 +121,9 @@ ge25519_scalarmult_base_choose_niels(ge25519_niels *t, uint32_t pos, signed char
 
 /* computes [s]basepoint */
 static void
+#if gpu_nvidia(DEVICE_INFO) && DEV_VER_MAJOR > 525 && DEV_VER_MAJOR < 560
```

Comment:
Again, you probably mean `>= 525`, not `> 525`.

Reply:
Oh, I see you wrote you saw the issue with 535 and 550. But apparently not with 525, nor 560. If so, the check is right.

Reply:
Generic, but very appropriate to the situation: it's documenting that anything after 525 and before 560 is or could be bad.